src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/ContainerTests.ps1

```diff
@@ -12,31 +12,61 @@
 # limitations under the License.
 # ----------------------------------------------------------------------------------
 
+$resourceGroupName = "labRG1";
+$resourceName = "pstestrsvault";
+$vaultLocation = "westus";
```

Comment:
We've tried to use Get-AzureLocation but it has some problem during playback in the Dogfood environment - where we are currently testing. The test fails with "Subscription ID is not present in the recorded mock or in environment variables." in playback mode; I've noticed that these calls are not even recorded in the json files. We've witnessed a similar issue with making Storage cmdlet calls before. So, we can't take this change unless this issue is fixed.